libs/add-child/feature/src/lib/add-child.component.html

```diff
@@ -0,0 +1,39 @@
+<ion-content>
+    <the-au-pair-navbar></the-au-pair-navbar>
+    <h2>Add A child</h2>
+    <div class="childrenDetails">
+        <form #childDetailsForm="ngForm" (ngSubmit)="getChildValues(childDetailsForm.value)" class = "form-details">
+
+          <!-- Child First Name -->
+          <label for="childName" style="left: -156px !important;">Child Name</label>
+          <input type="text" ngModel name="childName"placeholder="Child Name">
+          <p id="childNameError" class="error" style="display: none;">Error in field</p>
+
+          <!-- Surname -->
+          <label for="surname" style="left: -165px !important;">Surname</label>
+          <input type="text" ngModel name="surname" placeholder="Surname"> 
+          <p id="surnameError" class="error" style="display: none;">Error in field</p>
```

Comment:
Should these errors be a bit more descriptive?

Reply:
The inner HTML of these error messages are checked and set in the component.ts file and are more descriptive when specific errors are encountered.  That is just a default for miscellaneous errors.